### PR TITLE
Pixelated image rendering

### PR DESCRIPTION
**Pull request by paulinevos**
Currently image rendering is the browser default (`auto`) which is anti-aliased. This isn't ideal for annotations as we want to be able to annotate on raw data.

This PR introduces a pixelated default, with the option to enable anti-aliasing if desired. Applies to the old UI (static, jQuery) as well as the new UI (React, in case we start using that in the future).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,5 +24,5 @@ __pycache__
 
 # Ignore development npm files
 node_modules
-
+.idea
```
Comment (ryushkou) [inline .gitignore]: No need in this local env specific. Please remove.

File: .gitignore [hunk 2]
```diff
@@ -14,6 +14,7 @@
 !/ssh/README.md
 node_modules
 /Mask_RCNN/
+cvat-ui/react_nginx.conf
```
Comment (ryushkou) [inline .gitignore]: No need in this local env specific. Please remove.

File: cvat-ui/.gitignore [hunk 3]
```diff
@@ -4,4 +4,5 @@
 /dist
 /build
 /yarn.lock
+/.idea
```
Comment (ryushkou) [inline cvat-ui/.gitignore]: No need in this local env specific. Please remove.

**Comment by ryushkou** [comment]
Added comments for .gitignore changes

File: .gitignore (hunk 2 above)
Comment (paulinevos) [reply .gitignore]: I don't understand, isn't that exactly why you'd _want_ it in .gitignore and have a `.dist` version in the repo instead?

Comment (paulinevos) [reply .gitignore]: We also discussed it on the call

Comment (ryushkou) [reply .gitignore]: Agreed to handle it in the scope of another ticket.

**Review by PDLdeLange — APPROVED** [review]

**Review by ryushkou — APPROVED** [review]

**Pull request merged**